.github/workflows/dependencies.yml

```diff
@@ -66,8 +68,10 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v7
         with:
+          token: ${{ secrets.STOROPOLI_CREATE_PR_TOKEN }}
```

Comment:
This looks like a bad idea. Probably best to have a separate token for github actions (and named as such).

Reply:
We had a brief discussion about the token here: https://alpenlabs.atlassian.net/browse/STR-676

Reply:
Replied to the ticket. But the idea is to basically use a separate account, or a token managed by the repo/org owners. Using a personal token should really be our last resort if all else fails.

Reply:
So the restriction on PRs created from within this repo by github actions has been lifted?

Reply:
AFAIK, you need to add the permissions as I did in https://github.com/alpenlabs/strata/blob/5bb20c42b7ba6fc395af675d61720aa2ec218276/.github/workflows/dependencies.yml#L32-L34